docs/content/doc/usage/email-setup.en-us.md

```diff
@@ -15,15 +15,27 @@ menu:
 
 # Email setup
 
-- To use Gitea's built-in Email support, update the `app.ini` config file [mailer] section:
+To use Gitea's built-in Email support, update the `app.ini` config file [mailer] section:
 
+- Sendmail version - Use the operating system’s sendmail command instead of SMTP. This is common on linux servers.
 ```ini
 [mailer]
-ENABLED = true
-HOST    = mail.mydomain.com:587
-FROM    = gitea@mydomain.com
-USER    = gitea@mydomain.com
-PASSWD  = `password`
+ENABLED       = true
+FROM          = gitea@mydomain.com
+MAILER_TYPE   = sendmail
+SENDMAIL_PATH = sendmail
```

Comment:
```suggestion
SENDMAIL_PATH = /usr/sbin/sendmail
```
Small nit just think it is better to always use full paths in examples (and in general) for clarity